nengo/connection.py

```diff
@@ -319,9 +317,10 @@ def __init__(self, pre, post, synapse=Default, transform=Default,
         self.function_info = function  # Must be set after transform
         self.solver = solver  # Must be set before learning rule
         self.learning_rule_type = learning_rule_type  # set after transform
-        self.seed = seed
         self.modulatory = modulatory
 
+        super(Connection, self).__init__(label=label, seed=seed)
```

Comment:
Is there some reason this is called at the end? Usually I put it at the beginning if it doesn't matter, but that's just personal preference.